### PR TITLE
Support DF S3 and S4 tier sets for enhancement

### PR DESCRIPTION
**Pull request by Seriousnes**
### Description

* Adding support for S4 tier set
* Earth Shield mana cost was wrong
* Frost Mage changelog was using wrong format

### Testing

- Test report URL: `/report/mjypMY7QtC6TLP1w/17-Mythic+Fyrakk+the+Blazing+-+Kill+(9:26)/2-Seriousnes/standard`

### CODE REVIEW

File: src/analysis/retail/mage/frost/CHANGELOG.tsx [hunk 1]
```diff
@@ -6,7 +6,7 @@ import { Earosselot, Sharrq, Sref, ToppleTheNun } from 'CONTRIBUTORS';
 
 // prettier-ignore
 export default [
-  change(date(2024, 20, 4), <>Improved <SpellLink spell={SPELLS.WINTERS_CHILL} /> analysis on guide. Setting guide as default view. </>, Earosselot),
```
Comment (Seriousnes) [inline src/analysis/retail/mage/frost/CHANGELOG.tsx:9]: This is showing as 04/08/2025, which I don't believe to be true unless @earosselot is a timetraveller 🤣

File: src/common/TALENTS/shaman.ts [hunk 2]
```diff
@@ -349,7 +349,7 @@ const talents = {
     maxRanks: 1,
     entryIds: [102004],
     definitionIds: [{ id: 106968, specId: 262 }],
-    manaCost: 5000,
+    manaCost: 12500,
```
Comment (emallson) [inline src/common/TALENTS/shaman.ts:352]: this is going to be overwritten by the next generation, but maybe that is fine if the mana cost has been updated in-game and the next generation just hasn't happened yet?

Comment (Seriousnes) [reply src/common/TALENTS/shaman.ts:352]: I just noticed the mana cost warning in console, not a big deal if it gets overridden.

Comment (emallson) [reply src/common/TALENTS/shaman.ts:352]: kk

**Pull request merged**